internal/cli/atlas/clusters/advancedsettings/update.go

```diff
@@ -0,0 +1,194 @@
+// Copyright 2022 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package advancedsettings
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/mongodb/mongodb-atlas-cli/internal/cli"
+	"github.com/mongodb/mongodb-atlas-cli/internal/cli/atlas/commonerrors"
+	"github.com/mongodb/mongodb-atlas-cli/internal/cli/require"
+	"github.com/mongodb/mongodb-atlas-cli/internal/config"
+	"github.com/mongodb/mongodb-atlas-cli/internal/flag"
+	"github.com/mongodb/mongodb-atlas-cli/internal/store"
+	"github.com/mongodb/mongodb-atlas-cli/internal/usage"
+	"github.com/openlyinc/pointy"
+	"github.com/spf13/cobra"
+	atlas "go.mongodb.org/atlas/mongodbatlas"
+)
+
+const (
+	updateTmpl                   = "Updating advanced configuration settings of your cluster'.\n"
+	defaultReadConcern           = "available"
+	defaultWriteConcern          = "1"
+	defaultSampleSizeBIConnector = 1000
```

Comment:
I'm catching just now that I think these should be removed, you are doing and if this value then don't set it for the request, which means if I change the read concern and want to go back to the default I can't, right?

Reply:
yes, good catch!

Reply:
updated